<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Peter Humburg" />
  <title>SNPs, ChIPs and RNA</title>
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
  <link rel="stylesheet" href="reveal.js/css/reveal.min.css"/>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="reveal.js/css/theme/sydney.css" id="theme">
    <!-- If the query includes 'print-pdf', include the PDF print sheet -->
    <script>
      if( window.location.search.match( /print-pdf/gi ) ) {
        var link = document.createElement( 'link' );
        link.rel = 'stylesheet';
        link.type = 'text/css';
        link.href = 'reveal.js/css/print/pdf.css';
        document.getElementsByTagName( 'head' )[0].appendChild( link );
      }
    </script>
    <!--[if lt IE 9]>
    <script src="reveal.js/lib/js/html5shiv.js"></script>
    <![endif]-->
    <script src="/MathJax/MathJax.js?config=TeX-AMS-MML_HTMLorMML,local/local" type="text/javascript"></script>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">SNPs, ChIPs and RNA</h1>
  <h1 class="subtitle">Attempts to understand complex phenotypes</h1>
    <h2 class="author">Peter Humburg</h2>
    <h3 class="date">15th April 2015</h3>
</section>
<section id="TOC">
<h1>Overview</h1>
<ul>
<li><a href="#/introduction">Introduction</a></li>
<li><a href="#/exome-sequencing">Exome sequencing</a></li>
<li><a href="#/non-coding-variants-more-information-needed">Non-coding variants: More information needed</a></li>
<li><a href="#/conclusions">Conclusions</a></li>
</ul>
</section>

<section><section id="introduction" class="titleslide slide level1"><h1>Introduction</h1></section><section id="interpreting-genomic-variation" class="slide level2">
<h1>Interpreting genomic variation</h1>
<ul>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Sequencing of patient genomes increasingly common</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Can identify relevant variants</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="1"> … amongst a large number of unrelated variants</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="1"> Computational strategies can narrow the set of candidates</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="2"> … but variants are often difficult to interpret</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="2"> Want to leverage existing data as much as possible</span></li>
</ul>
<aside class="notes">
<p>Typical sequencing studies either focus on individual patients (or trios) or larger cohorts.</p>
<p>Obviously these serve different objectives, either individual diagnosis/treatment or generally gaining better understanding of phenotype.</p>
<p>Variants of interest may be rare or common.</p>
Understanding the impact of exonic variation is relatively easy (but not without pitfalls). Non-coding variation is much harder.
</aside>
</section></section>
<section><section id="exome-sequencing" class="titleslide slide level1"><h1>Exome sequencing</h1></section><section id="breast-and-ovarian-cancer-risk-genes" class="slide level2 small-header">
<h1>Breast and Ovarian Cancer risk genes</h1>
<div class="twocolumn">
<ul>
<li>Several DNA repair genes implicated in breast and ovarian cancer susceptability.</li>
<li>Strong evidence that rare loss-of-function variants confer increased risk.</li>
</ul>
<div class="fragment">
<ul>
<li>Sequenced exons of 507 DNA repair genes in 1,150 patients.</li>
<li>Sequenced pools of 24 individuals.</li>
<li>Included 79 individuals with known mutations in breast cancer predisposition genes as positive controls.</li>
</ul>
</div>
</div>
<aside class="notes">
Note that no controls were sequenced and samples are pooled (no barcoding) to reduce amount of time and money required.
</aside>
</section><section id="analysis-strategy" class="slide level2">
<h1>Analysis strategy</h1>
<ul>
<li>Sequence pools with HiSeq2000</li>
<li><span class="math">\(\gt\)</span> 480<span class="math">\(\times\)</span> coverage in 90% of target region</li>
<li>Call variants in pools with <a href="http://sourceforge.net/projects/syzygy/">Syzygy</a>
<ul>
<li>Good sensitivity for rare variants<br /> (24/26 SNPs and 51/54 indels)</li>
<li>Identified 34,564 variants</li>
</ul></li>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Functional annotation obtained via EnsEMBL</span>
<ul>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Substantial clean-up and curation of annotations</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Focused on 1,044 protein truncating variants</span></li>
</ul></li>
</ul>
</section><section id="variant-annotation" class="slide level2">
<h1>Variant annotation</h1>
<ul>
<li>Variant annotations depend on quality of transcript annotations.</li>
<li>Different annotation software may give different results.</li>
</ul>
<div class="fragment current-visible popup">
<img src="figure/annotation_paper.png" />
</div>
<div class="fragment current-visible popup">
<img src="figure/annotation.png" />
</div>
</section><section id="results" class="slide level2">
<h1>Results</h1>
<ul>
<li>Identified genes enriched for PTVs.</li>
<li>Strongest PTV enrichment observed for PPM1D.</li>
<li>Observed clustering of variants in final exon.</li>
</ul>
<div class="fragment">
<p>Sequenced affected exon in 7,781 cases and 5,861 controls</p>
<ul>
<li>18 PTVs in 6,912 breast cancer cases</li>
<li>12 PTVs in 1,121 ovarian cancer cases</li>
<li>1 PTV in controls</li>
</ul>
</div>
<div class="fragment current-visible popup">
<img src="figure/ppm1d_cluster.png" />
</div>
<aside class="notes">
<ul>
<li>Removed variants used as positive controls.</li>
<li>Five individuals had PPM1D PTVs in discovery cohort.</li>
<li>Note that position in transcript suggests these variants will escape NMD (may have been excluded by strict LoF filter)</li>
<li>25 PTVs total</li>
<li>Relative risk: BC 2.7, OV 11.5</li>
</ul>
</aside>
</section><section id="ppm1d" class="slide level2">
<h1>PPM1D</h1>
<ul>
<li>Induced in p53 dependent manner</li>
<li>Contributes to deactivation of p53</li>
<li>Part of negative feedback loop required to escape p53-dependent cell cycle arrest</li>
</ul>
<div class="fragment">
<ul>
<li>Truncated proteins show increased activity.</li>
<li>PPM1D over-expression previously associated with breast cancer.</li>
</ul>
<div class="fragment popup">
<img src="figure/ppm1d_paper.png" />
</div>
</div>
</section></section>
<section><section id="non-coding-variants-more-information-needed" class="titleslide slide level1"><h1>Non-coding variants: More information needed</h1></section><section id="interpreting-non-coding-variants" class="slide level2 small-header">
<h1>Interpreting non-coding variants</h1>
<div class="twocolumn">
<ul>
<li>Impact of non-coding variants unclear</li>
<li>Affected genes not obvious</li>
<li>Effect on gene expression typically unknown</li>
<li><span class="fragment highlight-current-red" data-fragment-index="0"> Existing epigenetic data may help</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="1"> eQTL analyses can help to establish links between SNPs and genes</span></li>
</ul>
<div class="fragment current-visible overlap" data-fragment-index="0">
<img src="figure/encode.png" /> <img src="figure/blueprint.jpg" />
</div>
<div class="fragment current-visible overlap" data-fragment-index="1">
<img src="figure/eqtl_network.png" />
</div>
</div>
</section><section id="using-additional-genome-wide-data" class="slide level2 small-header">
<h1>Using additional genome-wide data</h1>
<p>ChIP-seq and RNA-seq data provide insight into the functional implications of genotypes. <strong>But</strong> need to consider</p>
<ul>
<li>Relevant cell type</li>
<li>Relevant conditions</li>
</ul>
<div class="fragment popup2" data-fragment-index="2">
<img src="figure/treatment_eqtl_paper.png" />
</div>
<div class="fragment popup" data-fragment-index="1">
<img src="figure/cell_type_eqtl_paper.png" />
</div>
</section><section id="the-role-of-rs11074938" class="slide level2 small-header">
<h1>The role of rs11074938</h1>
<div class="twocolumn">
<ul>
<li><a href="http://www.ensembl.org/Homo_sapiens/Variation/Explore?db=core;r=16:10912624-10912751;v=rs11074938;vdb=variation;vf=7436168">SNP</a> (A/G) located in intron of CIITA</li>
<li>Inside regulatory region</li>
<li>ENCODE data shows DHS and NF-<span class="math">\(\kappa\)</span>B binding</li>
<li>CIITA is important regulator of MHC class II expression</li>
<li>Could have implications for immune related diseases<br /> (<em>if this SNP affects CIITA</em>)</li>
</ul>
<div class="fragment" data-fragment-index="0">
<img src="figure/CIITA_encode.png" />
</div>
</div>
<div class="fragment fade-out popup" data-fragment-index="0">
<img src="figure/rs11074938.png" />
</div>
<div class="fragment popup">
<img src="figure/CIITA_paper.png" />
</div>
<aside class="notes">
<p>Annotations suggest SNP may have functional significance.</p>
<p>ENCODE data shows evidence of NF-<span class="math">\(\kappa\)</span>B in LCLs and DHS in LCLs and primary B cells (but not monocytes)</p>
Unclear which gene is affected and to what extend.
</aside>
</section><section id="changes-to-gene-expression" class="slide level2 small-header">
<h1>Changes to gene expression</h1>
<div class="fragment fade-out popup" data-fragment-index="0">
<img src="figure/cell_type_eqtl_paper.png" />
</div>
<div class="twocolumn">
<ul>
<li><span class="fragment highlight-current-red" data-fragment-index="1"> eQTL analysis in B cells shows reduced expression of CIITA associated with G allele</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="2"> Evidence for changes in expression of CIITA target genes</span></li>
<li><span class="fragment highlight-current-red" data-fragment-index="3"> Reduced presence of MHC class II proteins on cell surface associated with G allele</span></li>
</ul>
<div class="fragment current-visible overlap" data-fragment-index="1">
<img src="figure/CIITA_expression.png" />
</div>
<div class="fragment overlap current-visible" data-fragment-index="2">
<img src="figure/CIITA_network.png" />
</div>
<div class="fragment overlap current-visible" data-fragment-index="3">
<img src="figure/MHC_protein.png" />
</div>
</section><section id="transcription-factor-binding" class="slide level2 small-header">
<h1>Transcription factor binding</h1>
<div class="twocolumn">
<ul>
<li>Reduced NF-<span class="math">\(\kappa\)</span>B binding due to sequence change plausible</li>
<li>Used existing ChIP-seq data<span class="math">\(^*\)</span> to assess allele specific binding</li>
</ul>
<div>
<img src="figure/CIITA_asb.png" />
</div>
<div class="footnote">
<span class="math">\(^*\)</span> Kasowski <em>et al.</em>, Science (2010). PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20299548?dopt=Abstract">20299548</a>
</div>
</div>
</section><section id="does-it-matter" class="slide level2">
<h1>Does it matter?</h1>
<ul>
<li>No known GWAS associations</li>
<li>Good to see that available data and methods can identify functional non-coding SNPs.</li>
</ul>
<div class="fragment">
<ul>
<li>Some evidence for association with the presence of nasal polyps in asthma patients<span class="math">\(^*\)</span></li>
</ul>
<div class="footnote">
<span class="math">\(^*\)</span> Bae <em>et al.</em>, Mol Med Rep (2013). PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23292525?dopt=Abstract">23292525</a>
</div>
</div>
</section></section>
<section><section id="conclusions" class="titleslide slide level1"><h1>Conclusions</h1></section><section id="this-could-be-easier" class="slide level2">
<h1>This could be easier…</h1>
<ul>
<li>Functional annotation of genomic variants is still difficult.</li>
<li>Coding variants should be relatively easy, but annotations can be unreliable.</li>
<li>Non-coding variants are still difficult.</li>
</ul>
<div class="fragment">
<ul>
<li>Understanding the effect of genomic variation requires a lot of work.</li>
<li>Generating the data is easier than understanding it.</li>
</ul>
</div>
</section><section id="challenges-ahead" class="slide level2">
<h1>Challenges ahead</h1>
<ul>
<li>Data volume is increasing, can the analysis keep pace?</li>
<li>More and more public data sets available. Are we using them as much as we could?</li>
<li>Better integration of all types of genomic data.</li>
</ul>
<aside class="notes">
<p>If we want to analyse patient genomes on a large scale the analysis needs to be completed on time scales similar to the data generation.</p>
<p>This requires reliable, automated procedures.</p>
<p>Good quality data are available, some from large consortia, some from smaller studies.</p>
<p>Identifying the data that are most relevant to a given project isn’t always easy.</p>
Should systematically incorporate existing data into the analysis to provide additional context.
</aside>
</section><section id="acknowledgments" class="slide level2 acknowledge small-header">
<h1>Acknowledgments</h1>
<h3 id="breast-cancer-risk-and-variant-annotation">Breast cancer risk and variant annotation</h3>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><img src="figure/peter-donnelly.jpg" /></td>
<td style="text-align: left;"><strong>Peter Donnelly</strong></td>
<td style="text-align: left;"><img src="figure/nazneen-rahman.jpg" /></td>
<td style="text-align: left;"><strong>Nazneen Rahman</strong></td>
</tr>
<tr class="even">
<td style="text-align: left;"><img src="figure/manuel-rivas.jpg" /></td>
<td style="text-align: left;">Manuel A. Rivas</td>
<td style="text-align: left;"><img src="figure/katie-snape.jpg" /></td>
<td style="text-align: left;">Katie Snape</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><img src="figure/andrew-rimmer.jpg" /></td>
<td style="text-align: left;">Andrew Rimmer</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Elise Ruark</td>
</tr>
<tr class="even">
<td style="text-align: left;"><img src="figure/davis-mccarthy.jpeg" /></td>
<td style="text-align: left;">Davis McCarthy</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</section><section id="acknowledgments-1" class="slide level2 acknowledge small-header">
<h1>Acknowledgments</h1>
<h3 id="eqtl-and-ciita">eQTL and CIITA</h3>
<div class="twocolumn">
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><img src="figure/julian-knight.jpg" /></td>
<td style="text-align: left;"><strong>Julian C. Knight</strong></td>
</tr>
<tr class="even">
<td style="text-align: left;"><img src="figure/daniel-wong.jpg" /></td>
<td style="text-align: left;">Daniel Wong</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Wanseon Lee</td>
</tr>
<tr class="even">
<td style="text-align: left;"><img src="figure/ben-fairfax.jpg" /></td>
<td style="text-align: left;">Benjamin P. Fairfax</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><img src="figure/seiko-makino.jpg" /></td>
<td style="text-align: left;">Seiko Makino</td>
</tr>
</tbody>
</table>
<div id="group-photo">
<img src="figure/knightlab.jpg" />
</div>
</div>
</section></section>
    </div>
  </div>


  <script src="reveal.js/lib/js/head.min.js"></script>
  <script src="reveal.js/js/reveal.min.js"></script>

  <script>

      // Full list of configuration options available here:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        controls: true,
        progress: true,
        history: true,
        center: true,
        slideNumber: true,
        theme: 'sydney', // available themes are in /css/theme
        transition: Reveal.getQueryHash().transition || 'default', // default/cube/page/concave/zoom/linear/fade/none

        // Optional libraries used to extend on reveal.js
        dependencies: [
          { src: 'reveal.js/lib/js/classList.js', condition: function() { return !document.body.classList; } },
          { src: 'reveal.js/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
          { src: 'reveal.js/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
//          { src: 'reveal.js/plugin/search/search.js', async: true, condition: function() { return !!document.body.classList; }, }
//          { src: 'reveal.js/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
]});
//	Reveal.configure({slideNumber:true});
    </script>
    </body>
</html>
