---
title: "SNPs, ChIPs and RNA: Attempts to understand complex phenotypes"
author: Peter Humburg
---

# Interpreting genomic variation

* Sequencing of patient genomes increasingly common
* Can identify relevant variants

. . .

* ... amongst a large number of unrelated variants
* Computational strategies can narrow the set of candidates

. . .

* ... but non-coding variants are difficult to interpret
* Want to leverage existing data as much as possible 

<div class="notes">
Typical sequencing studies either focus on individual patients (or trios)
or larger cohorts.

Obviously these serve different objectives, either individual diagnosis/treatment
or generally gaining better understanding of phenotype.

Variants of interest may be rare or common.

Understanding the impact of exonic variation is relatively easy (but not without
pitfalls). Non-coding variation is much harder.
</div>

# Exome sequencing
## Identifying novel Breast and Ovarian Cancer risk genes
<div class="twocolumn">
	
* Several DNA repair genes implicated in breast and ovarian cancer susceptability.
* Strong evidence that rare loss-of-function variants confer increased risk.

<div class="fragment">
* Sequenced exomes of 507 DNA repair genes in 1,150 patients.
* Sequenced pools of 24 individuals.
* Included 79 individuals with known mutations in breast cancer predisposition 
  genes as positive controls. 
* Replicated findings in 7,781 cases and 5,861 controls.

<div>
<div class="fragment">
	![](figure/ppm1d_paper.png)
</div>
</div>

## Analysis strategy

* 
 

# Non-coding variants: More information needed